common/action-manager.js

```diff
@@ -53,7 +53,8 @@ const am = new ActionManager([
   "SHARE_PROVIDERS_REQUEST",
   "SHARE_PROVIDERS_RESPONSE",
   "TOP_FRECENT_SITES_REQUEST",
-  "TOP_FRECENT_SITES_RESPONSE"
+  "TOP_FRECENT_SITES_RESPONSE",
+  "SYNC_COMPLETE"
```

Comment:
Bonus points if we sort this alphabetically, so @k88hudson won't flip a table (ref #1837).